src/main/java/com/segment/analytics/android/integrations/comscore/ComScoreIntegration.java

```diff
@@ -235,6 +240,16 @@ private void trackVideoPlayback(
     if (playbackPosition == 0) {
       playbackPosition = properties.getLong("position", 0);
     }
+    String adType = properties.getString("adType");
+    if (adType == null || adType.trim().isEmpty()) {
```

Comment:
Why do we check `isEmpty` in the first conditional, but not in the next?

Reply:
Updated conditional checks in commit https://github.com/segment-integrations/analytics-android-integration-comscore/pull/18/commits/028ab7c132349591686a791e7713a474b87d6bed